src/openklant/utils/query.py

```diff
@@ -17,5 +17,7 @@ def get_schema_operation_parameters(self, auto_schema, *args, **kwargs):
 
         for parameter in parameters:
             parameter["name"] = underscore_to_camel(parameter["name"])
+            if parameter["name"].endswith("__url"):
+                parameter["schema"] = {"type": "string", "format": "uri"}
```

Comment:
@SonnyBA I discussed this with @bart-maykin, because the underlying field that is referred to is a `uuid`, drf-spectacular adds `format: uuid` for all these fields in the OAS. It's not possible to set the format via the filter class, so that leaves us with this workaround or to add `extend_schema_field` for all fields and override the format (and add the help text). What do you think?